### DISCUSSION TITLE
Remuxing A Stream

### DISCUSSION THREAD

**Discussion by MattWanjia** [bluenviron/mediamtx#3661] [Questions]
### Question

I am trying to repackage my streams from rtsp coming in MPEGTS format to h264 format. The code is below.
I am expecting my stream at the url attached but not getting a stream. 
Error: NO ONE IS PUBLISHING TO THAT PATH.
am i doing anything wrong?  

startup:
    runOnInit: curl -s http://localhost:8081/api/mediamtx/webhook?path=$MTX_PATH&rtsp_port=$RTSP_PORT&event=init&token=SRFAWSGRujxO5djMuxm5IgmJAAOXwQ > /dev/null
  # example:
  # my_camera:
  #   source: rtsp://my_camera

  # Settings under path "all_others" are applied to all paths that
  # do not match another entry.
  all_others:
  compressed:
  #original:
    runOnReady: >
      ffmpeg -i rtsp://localhost:$RTSP_PORT/$MTX_PATH
        -pix_fmt yuv420p -c:v libx264 -preset ultrafast -b:v 600k
        -max_muxing_queue_size 1024 -f rtsp rtsp://localhost:$RTSP_PORT/compressed/$MTX_PATH
    runOnReadyRestart: yes

**Comment by aler9** [comment]
As written in the error message, no stream is available on the path you are trying to read. At the same time, you didn't provide any proof that you are actually publishing something to the server, on which path and with what software.

If you still have issues, open another discussion, provide clear instructions on how to replicate the issue and attach server logs.
